src/test/scala/nl.knaw.dans.easy.properties/server/GraphQLResolveSpec.scala

```diff
@@ -1530,7 +1530,7 @@ class GraphQLResolveSpec extends TestSupportFixture
         depositId2 -> step2,
         depositId3 -> step3,
       ).asRight
-      depositDao.search _ expects Seq(filters3, filters2, filters4) once() returning Seq(
+      depositDao.search _ expects Seq(filters2, filters3, filters4) once() returning Seq(
```

Comment:
You still use numbers here, please use meaningful names. E.g. validationStep, FedoaraStep etc.